PCGen-Formula/code/src/java/pcgen/base/formula/visitor/StaticVisitor.java

```diff
@@ -118,8 +118,7 @@ public StaticVisitor(FunctionLibrary fl)
 	 * 
 	 * {@inheritDoc}
 	 */
-	@Override
-	public Object visit(SimpleNode node, Object data)
+	public Object visit(SimpleNode node)
```

Comment:
We don't control the interface, it is automatically generated by javacc... and it's not something we check in.